src/main/java/mil/dds/anet/beans/FileUpload.java

```diff
@@ -0,0 +1,40 @@
+package mil.dds.anet.beans;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.leangen.graphql.annotations.GraphQLInputField;
+import io.leangen.graphql.annotations.GraphQLQuery;
+
+import java.io.File;
+
+public class FileUpload {
```

Comment:
Seems to me like this maps the attachment table, so the class should be called Attachment? It also misses a.o. the description field of the attachment table.